doc_source/GettingStartedTutorial.md

```diff
@@ -92,7 +92,7 @@ Now that you have created your Auto Scaling group, you are ready to verify that
 
    1. On the navigation pane, choose **Auto Scaling Groups**\. Select your Auto Scaling group and chose the **Activity History** tab\. The default cooldown for the Auto Scaling group is 300 seconds \(5 minutes\), so it takes about 5 minutes until you see the scaling activity\. When the scaling activity starts, you'll see an entry for the termination of the first instance and an entry for the launch of a new instance\. The **Instances** tab shows the new instance only\.
 
-   1. On the navigation pane, choose **Instances**\. This page shows both the terminated instance and the running instance\.
+   1. On the navigation pane, choose **Activity History**\. This page shows both the terminated instance and the running instance\.
 
```

Comment:
Good catch.

Reply:
I tested this and the documentation matches the user interface. Perhaps this is an issue of terminology. The navigation pane is the left-hand panel; the Instances item appears in this pane under INSTANCES. Activity History is one of the tabs you can see after you select Auto Scaling Groups from the navigation pane and select an Auto Scaling group. I hope this helps.

Reply:
Got it. Thanks for pointing that out. I didn't read it carefully enough.